src/components/V3Select.json

```diff
@@ -0,0 +1,61 @@
+{
+    "V3Select": {
+      "fontFamily": {"value": "font.family.sans.value"},                          /* globalTokens.type_sans, */
```

Comment:
When we're referencing another token the value should be between curly brackets ( `"{value}"` )

Like this: `"fontFamily": {"value": "{font.family.sans.value}"},`

You can find more info in the [reference/aliasing section of style dictionary documentation](https://amzn.github.io/style-dictionary/#/tokens?id=referencing-aliasing)